src/engine/enginebuffer.cpp

```diff
@@ -253,9 +253,11 @@ EngineBuffer::EngineBuffer(const QString& group,
     m_pScale->clear();
     m_bScalerChanged = true;
 
-    m_pPassthroughEnabled = new ControlProxy(group, "passthrough", this);
-    m_pPassthroughEnabled->connectValueChanged(this, &EngineBuffer::slotPassthroughChanged,
-                                               Qt::DirectConnection);
+    m_pPassthroughEnabledControl = new ControlProxy(group, "passthrough", this);
+    m_pPassthroughEnabledControl->connectValueChanged(this,
+            &EngineBuffer::slotPassthroughChanged,
+            Qt::DirectConnection);
+    m_passthroughEnabled = false;
```

Comment:
Why `false` instead of `m_pPassthroughEnabledControl->toBool()`

Reply:
The EngineDeck constructor creates the "passthrough" ControlPushButton and instantiates EngineBuffer, so "passthrough" is still false here.
Anyway, you're right, someone may change it so "passthrough" is initiated `true` then this wouldn't match.